### PR TITLE
Task-1, Task-2 & Task-3 Complete with Error and Bug fixing.

### PR DESCRIPTION
**Pull request by jishnujayakumar**
Every error known to me is taken care of. The code is running fine. There was a bug in row filtering code and it has been fixed by me and now it's running well. So merge this pull request.

### CODE REVIEW

File: .metadata/.log [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+!SUBENTRY 1 org.eclipse.equinox.p2.operations 2 10004 2017-07-07 10:25:54.826
+!MESSAGE "PyDev for Eclipse" will be ignored because a newer version is already installed. 
+!SUBENTRY 1 org.eclipse.equinox.p2.operations 2 10004 2017-07-07 10:25:54.826
+!MESSAGE "PyDev for Eclipse Developer Resources" will be ignored because a newer version is already installed. 
+
+!ENTRY org.eclipse.ui 2 2 2017-07-07 10:26:33.812
+!MESSAGE Invalid property category path: ValidationPropertiesPage (bundle: org.eclipse.wst.html.ui, propertyPage: org.eclipse.wst.html.ui.propertyPage.project.validation)
+
+!ENTRY org.eclipse.ui 2 2 2017-07-07 10:26:33.812
+!MESSAGE Invalid property category path: ValidationPropertiesPage (bundle: org.eclipse.wst.xml.ui, propertyPage: org.eclipse.wst.xml.ui.propertyPage.project.validation)
```
Comment (anantharam) [inline .metadata/.log]: Can you please remove the.metadata files from the pull request?

File: .metadata/.plugins/org.eclipse.ui.workbench/workbench.xml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+<trimArea IMemento.internal.id="1024">
+<trimItem IMemento.internal.id="org.eclise.ui.internal.FastViewBar"/>
+<trimItem IMemento.internal.id="org.eclipse.jface.action.StatusLineManager"/>
+<trimItem IMemento.internal.id="org.eclipse.ui.internal.progress.ProgressRegion"/>
+<trimItem IMemento.internal.id="com.google.gdt.eclipse.login.status.trim"/>
+</trimArea>
+</trimLayout>
+</window>
+<mruList/>
+</workbench>
```
Comment (anantharam) [inline .metadata/.plugins/org.eclipse.ui.workbench/workbench.xml]: All of these shouldn't be in git

File: utils/phpserialize.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python -u
+
+"""
+Python port of PHP serialize() and unserialize()
+by Samuel Cochran <sj26@sj26.com>
+
```
Comment (anantharam) [inline utils/phpserialize.py]: Why exactly do we need this file?

Comment (jishnujayakumar) [inline utils/phpserialize.py]: Actually, ubuntu and python have generated this files. And they were hidden so I couldn't see them.

**Pull request merged**